crates/meili-snap/Cargo.toml

```diff
@@ -11,6 +11,6 @@ edition.workspace = true
 license.workspace = true
 
 [dependencies]
-insta = { version = "^1.39.0", features = ["json", "redactions"] }
+insta = { version = "=1.39.0", features = ["json", "redactions"] }
```

Comment:
Can we motivate this change?

Reply:
Yup, insta broke the file format between 1.39 and 1.40 (latest is 1.41) and I don't know why...

Reply:
```suggestion
# fixed version due to format breakages in v1.40
insta = { version = "=1.39.0", features = ["json", "redactions"] }
```